cypress.config.js

```diff
@@ -1,9 +1,11 @@
 const { defineConfig } = require('cypress');
-
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com',
     setupNodeEvents(on, config) {
-      // implement node event listeners here
+      on('task', {
```

Comment:
```suggestion
      on('task', {
```
why have you added this?